# This message is intended primarily for debugging and visualisation purposes.
# It contains raw data from detected blobs.

uint32  x
uint32  y
float32 avg_depth
float32 confidence
float32 convexity
float32 angle
uint32  area
float32 circularity
float32 radius
float32 inertia
Contour[] contours
